### PR TITLE
Patroni: correction from copy edit - code block adjustments

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

Clarified edit related to listening on ports
Removed prompt from code blocks so that the contents can be copied and pasted in a terminal and work

### CODE REVIEW

**Comment by martinmarques** [comment]
Dee Dee can the same code block be applied to the rest of the documentation before merging? Right now it only applies to "Managing a cluster" and not the the quick starts or "Patroni" and "Etcd" installation instructions

**Comment by drothery-edb** [comment]
@martinmarques I ran into two issues trying to apply the code block output formatting: you can't have commands in a code block after output and the __OUTPUT__ markup was not being interpreted correctly after cat << commands. For both of these issues I ended up splitting up the code blocks. I'm not sure if you will find the workaround acceptable. You can decide for yourself by looking at these topics after the draft deploy complete:

- https://deploy-preview-3892--edb-docs-staging.netlify.app/docs/supported-open-source/patroni/debian11_quick_start/
- https://deploy-preview-3892--edb-docs-staging.netlify.app/docs/supported-open-source/patroni/installing_etcd/

Let me know what you think.

File: advocacy_docs/supported-open-source/patroni/rhel8_quick_start.mdx [hunk 1]
```diff
@@ -316,22 +316,25 @@ $ curl -s http://pg-patroni1:8008
...
+psql -U admin -d postgres -h pg-patroni1 -p 5000 -c "SELECT pg_is_in_recovery();"
+__OUTPUT__
  pg_is_in_recovery
 -------------------
  f
 (1 row)
-
+```
+```bash
 $ psql -U admin -d postgres -h pg-patroni1 -p 5001 -c "SELECT pg_is_in_recovery();"
```
Comment (martinmarques) [inline advocacy_docs/supported-open-source/patroni/rhel8_quick_start.mdx:336]: Need to remove the `$` here as well

**Review by martinmarques — CHANGES_REQUESTED** [review]
Besides the missing removal of one `$`, we also need to apply the same to the Patroni installation instructions and the "Tips and FAQ"

File: advocacy_docs/supported-open-source/patroni/rhel8_quick_start.mdx (hunk 1 above)
Comment (djw-m) [reply advocacy_docs/supported-open-source/patroni/rhel8_quick_start.mdx]: Fixed in last commit

**Review by martinmarques — APPROVED** [review]

**Pull request merged**